packages/models/src/purpose/purpose.ts

```diff
@@ -48,6 +49,7 @@ export const Purpose = z.object({
   id: PurposeId,
   eserviceId: EServiceId,
   consumerId: TenantId,
+  delegationId: DelegationId.optional(),
```

Comment:
⚠️ new optional field added to a model - make sure you double check all the converters or points in the code where this could be easily forgotten 🥲 
Consequence of the infamous issue tracked here: https://pagopa.atlassian.net/browse/PIN-5598